jcl/src/java.base/share/classes/com/ibm/tools/attach/target/IPC.java

```diff
@@ -185,19 +183,23 @@ public static void checkOwnerAccessOnly(String filePath) throws IOException {
 
 	/**
 	 * Open the semaphore, post to it, and close it
+	 * @param ctrlDir directory containing the semaphore
+	 * @param semaphoreName name of the semaphore
 	 * @param numberOfTargets number of times to post to the semaphore
+	 * @param global open the semaphore in the global namespace (Windows only)
 	 * @return 0 on success
 	 */
-	native static int notifyVm(String ctrlDir, String SemaphoreName,
-			int numberOfTargets);
+	native static int notifyVm(String ctrlDir, String semaphoreName,
+			int numberOfTargets, boolean global);
 
 	/**
 	 * Open the semaphore, decrement it without blocking to it, and close it
 	 * @param numberOfTargets number of times to decrement to the semaphore
```

Comment:
Might as well add the remaining arguments to the doc, since you fixed notifyVM()

Reply:
Done.